net/ddns-scripts/files/usr/lib/ddns/update_gandi_net.sh

```diff
@@ -1,19 +1,37 @@
 #!/bin/sh
 # Thanks goes to Alex Griffin who provided this script.
 
+. /usr/share/libubox/jshn.sh
+
 local __TTL=600
 local __RRTYPE
 local __ENDPOINT="https://dns.api.gandi.net/api/v5"
+local __STATUS
 
 [ -z "$username" ] && write_log 14 "Service section not configured correctly! Missing subdomain as 'username'"
 [ -z "$password" ] && write_log 14 "Service section not configured correctly! Missing API Key as 'password'"
 
 [ $use_ipv6 -ne 0 ] && __RRTYPE="AAAA" || __RRTYPE="A"
 
-curl -s -X PUT "$__ENDPOINT/domains/$domain/records/$username/$__RRTYPE" \
+# Construct JSON payload
+json_init
+json_add_string rrset_ttl "$__TTL"
+json_add_array rrset_values "$__IP"
+ubus call network add_dynamic "$(json_dump)"
```

Comment:
You could delete this line.
This is not necessary.
I just wanted to show you where you can have a look how to use jshn.

Reply:
Oh, my bad, I thought I've deleted that part

Reply:
Please stash your commits to one after I have merged https://github.com/openwrt/packages/pull/16429 you should the bump the PKG_RELEASE number

Reply:
Certainly, I will test these changes for myself as well and rebase all of the commits.
So the `PKG_RELEASE` should be bumped for both?

Reply:
> Certainly, I will test these changes for myself as well and rebase all of the commits.
> So the `PKG_RELEASE` should be bumped for both?
 
Yes we need a PKG_RELEASE bump for this pullrequest and for the other one.

> 
> Edit: BTW, what's with the CI error for `x86_64`?

I have asked myself the same question. I'll have to ask the others. I haven't looked at it yet.